core/kazoo_number_manager/src/carriers/knm_other.erl

```diff
@@ -57,9 +57,9 @@ is_local() -> 'false'.
 %% in a rate center
 %% @end
 %%--------------------------------------------------------------------
--spec find_numbers(ne_binary(), pos_integer(), knm_carriers:options()) ->
-                          {'ok', knm_number:knm_numbers()} |
-                          {'bulk', knm_number:knm_numbers()} |
+-spec find_numbers(ne_binary(), pos_integer(), knm_search:options()) ->
+                          {'ok', list()} |
```

Comment:
Still wondering why you changed these to more generic versions. It breaks the behaviour's expected return:

```
knm_other.erl:60: The return type {'bulk',[any()]} | {'ok',[any()]} in the specification of find_numbers/3 is not a subtype of {'bulk',[knm_number:knm_number()]} | {'error',_} | {'ok',[knm_number:knm_number()]}, which is the expected return type for the callback of knm_gen_carrier behaviour
```